.github/workflows/linting.yml

```
@@ -0,0 +1,34 @@
+
+jobs:
+    Linter:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v4
+
+            - name: Set up Node.js
+              uses: actions/setup-node@v4
+              with:
+                node-version: '20'
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 18 but found 16

(indentation)